### PR TITLE
[ServiceBus] drop Python 2.7/3.6

### PR DESCRIPTION
**Pull request by swathipil**
fixes: #22265

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/CHANGELOG.md
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: should be 7.5.0

Comment (swathipil) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: updated in this PR: https://github.com/Azure/azure-sdk-for-python/pull/22317/commits/d6f5462dcbce29144ea620f1e6157515b6258faa.

will merge this one after the other PR is merged

File: sdk/servicebus/azure-servicebus/setup.py
Comment (rakshith91) [inline sdk/servicebus/azure-servicebus/setup.py:29]: why remove this?

Comment (swathipil) [reply sdk/servicebus/azure-servicebus/setup.py:29]: as per this: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/packaging.md#note-on-checking-old-azure-packages

File: sdk/servicebus/azure-servicebus/CHANGELOG.md
Comment (swathipil) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md:3]: edit: above PR merged

**Review by yunhaoling — APPROVED** [review]

**Pull request merged**
